casadi/core/std_vector_tools.hpp

```diff
@@ -744,6 +748,33 @@ namespace casadi {
     casadi_assert_message(0, "get_bvec_t only supported for double");
   }
 
+  template<typename T>
+  std::vector< std::vector<T> > transpose(const std::vector< std::vector<T> > &mat) {
```

Comment:
Maybe `swapIndices`? When I see `transpose`, I expect something transposing a matrix. But we never use `vector<vector<T> >` for representing matrices. When I saw this, I thought it would do: something along the lines of `[i.T for i in v]`.